packages/block/test/header.spec.ts

```diff
@@ -107,18 +107,6 @@ describe('[Block]: Header functions', () => {
       'block should not be frozen when freeze deactivated in options',
     )
 
-    assert.throws(
-      () =>
-        createHeaderFromRLP(rlpHeader, {
-          common,
-          freeze: false,
-          setHardfork: 1n, // Added to bypass defaulting setHardfork to true in static constructor
-        }),
-      'A base fee',
-      undefined,
-      'throws when RLP serialized block with no base fee on default hardfork (london) and setHardfork left undefined',
```

Comment:
This test is removed but I don't think this has anything to do with TTD, so we should likely keep this in?